src/common/transformations/src/transformations/common_optimizations/augru_cell_fusion.cpp

```diff
@@ -81,7 +81,11 @@ ov::pass::AUGRUCellFusion::AUGRUCellFusion() {
     // only Sigmoid is supported in the current version of AUGRUCell
     auto sigmoid = wrap_type<ov::op::v0::Sigmoid>({add_1});
     auto split = wrap_type<ov::op::v1::Split>({sigmoid, any_input()});
-    auto multiply = wrap_type<ov::op::v1::Multiply>({split, any_input()});
+    split->set_output_size(2);
```

Comment:
This looks strange to set number of outputs. 
Is there possibility to wrap Split type with number of splits?
Then number of outputs should be set.